packages/eslint-config-olo/rules/react.js

```diff
@@ -67,22 +75,29 @@ module.exports = {
     }],
     // Prevent React to be incorrectly marked as unused
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-react.md
-    'react/jsx-uses-react': [2, { 'pragma': 'React' }],
+    'react/jsx-uses-react': 2,
     // Prevent variables used in JSX to be incorrectly marked as unused
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-vars.md
     'react/jsx-uses-vars': 2,
+    // Prevent missing parentheses around multilines JSX
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/wrap-multilines.md
+    'react/jsx-wrap-multilines': [2, {
```

Comment:
It looks like this may conflict with Prettier, can we disable it in favor of using whatever Prettier outputs? https://github.com/prettier/prettier/issues/1009#issuecomment-286993938

Reply:
I don't think this conflicts -- the issue is more than two years old and was inconclusive. In my current experience there is no problem. If we consider adding prettier to this `javascript` project, there might be a conversation we can have at that time about where to place an appropriate rule.